### PR TITLE
feat: card tap input updated

### PR DESCRIPTION
**Pull request by AryanCypherock**
- updated card tap input 
- implemented it in Pin Recovery Dialog

### CODE REVIEW

File: packages/ui/src/components/molecules/CardTapList.tsx [hunk 1]
```diff
@@ -76,17 +81,26 @@ const CardTapListItem: React.FC<CardTapItemProps> = ({
     ));
 
   return (
-    <BackgroundStyle $bgColor={getBgColor(currentState, totalState)}>
+    <BackgroundStyle
+      $bgColor={
+        variant === 'other'
+          ? 'inputSecondary'
+          : getBgColor(currentState, totalState)
```
Comment (TejasvOnly) [inline packages/ui/src/components/molecules/CardTapList.tsx]: This logic should be inside the function itself

File: packages/ui/src/components/molecules/CardTapList.tsx [hunk 2]
```diff
@@ -76,17 +81,26 @@ const CardTapListItem: React.FC<CardTapItemProps> = ({
...
+    >
       <MaskStyle>
         <Typography
           variant="h6"
-          color={getTextColor(currentState, totalState)}
-          $fontWeight="medium"
+          color={
+            variant === 'other'
+              ? 'muted'
+              : getTextColor(currentState, totalState)
```
Comment (TejasvOnly) [inline packages/ui/src/components/molecules/CardTapList.tsx]: Logic should be inside the function

File: packages/ui/src/components/molecules/CardTapList.tsx [hunk 3]
```diff
@@ -76,17 +81,26 @@ const CardTapListItem: React.FC<CardTapItemProps> = ({
...
+          }
+          $fontWeight={variant === 'other' ? 'normal' : 'medium'}
```
Comment (TejasvOnly) [inline packages/ui/src/components/molecules/CardTapList.tsx]: Was the default font weight `'medium'`?

File: packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/WalletAuth.tsx [hunk 4]
```diff
@@ -123,6 +121,16 @@ export const WalletAuth = () => {
...
+              {
+                text: strings.walletAuth.actions.tapCard,
+                currentState: cardTapState,
+                totalState: 3,
+              },
+            ]}
+            variant="other"
+          />
         </LeanBoxContainer>
       </Container>
```
Comment (TejasvOnly) [inline packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/Dialogs/WalletAuth.tsx]:
```suggestion
        </LeanBoxContainer>
        <MessageBox
          type="warning"
          text={strings.wallet.walletAuth.messageBox.warning}
        />
      </Container>
```

File: packages/ui/src/components/molecules/CardTapList.tsx [hunk 5]
```diff
@@ -3,15 +3,19 @@ import { styled } from 'styled-components';
 
 import { Flex, Indicator, LangDisplay, Typography } from '../atoms';
 
+type VariantType = 'onboarding' | 'other';
```
Comment (TejasvOnly) [inline packages/ui/src/components/molecules/CardTapList.tsx]: It's better not to couple the parameter names with the flows
```suggestion
type VariantType = 'default' | 'muted';
```

**Review by TejasvOnly — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/molecules/CardTapList.tsx (hunk 3 above)
Comment (AryanCypherock) [reply packages/ui/src/components/molecules/CardTapList.tsx]: yes

**Review by TejasvOnly — APPROVED** [review]

**Pull request merged**